app/javascript/react/components/TrailShowContainer.js

```diff
@@ -15,6 +33,7 @@ const TrailShowContainer = (props) => {
             }
             const fetchedTrail = await response.json()
             setTrail(fetchedTrail)
+            getReviews()
```

Comment:
we will be deleting this in favor of getting reviews and the trail in one request